docs/integrations/destinations/bigquery-denormalized.md

```diff
@@ -8,8 +8,8 @@ See [destinations/bigquery](https://docs.airbyte.com/integrations/destinations/b
 
 | Version | Date       | Pull Request                                               | Subject                                                                                                                  |
 | :------ | :--------- | :--------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------- |
+| 1.3.0   | 2023-04-18 | [25287](https://github.com/airbytehq/airbyte/pull/25287)   | Add configurable parameter for number of file buffers                                                                                                      |
```

Comment:
Would add that this is isolated to using GCS as a loading method, both here and for regular `destination-bigquery`